src/components/search/results/TabViews.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { BsListUl, BsMap, BsCalendar3Week } from 'react-icons/bs';
+import EventList from './EventList';
+import styles from '../../../styles/resultList.module.css';
+
+const TabViews = () => {
+  const [view, setView] = useState(1);
+  const changeView = (index) => {
+    setView(index);
+  };
+
+  return (
+    <div className={styles.tabsContainer}>
+      <div className={styles.roundedTabs}>
+        <button
+          className={view === 1 ? styles.activeTabs : styles.tabs}
+          onClick={() => changeView(1)}>
+          <BsListUl />
+        </button>
+        <button
+          className={view === 2 ? styles.activeTabs : styles.tabs}
+          onClick={() => changeView(2)}>
+          <BsMap />
+        </button>
+        <button
+          className={view === 3 ? styles.activeTabs : styles.tabs}
+          onClick={() => changeView(3)}>
```

Comment:
There's a lot of repetitive code here that you could simplify down to something that looks like this:
```jsx
<TabContainer onSelect={setView}>
  <BsListUl />
  <BsMap />
  <BsCalendar3Week />
</TabContainer>
```
But you would need to create `TabContainer` yourself, unless some other library has already made one. Something to think about.